### ISSUE TITLE
when changing to a different priority with build menu open

### ISSUE THREAD

**Issue by donnti** [MaFi-Games/Captain-of-Industry-issues#204]
**Describe the issue**
If you have the build menu open and change to a different priority (P1), the cursor jumps to the search box when pressing the A or S key to move the camera. It types it in the search box.

**To Reproduce**
Open any build menu item (Metallurgy), then build something (Blast furnace); after building, change the priority to P1 with the build menu still open. When you move the camera to the left or down, it does not move, and the cursor is typing into the search box.

**Expected behavior**
The camera should move when you want it to.

**Additional context**
 - Game version: 0.4.1d

**Attachments**
I did make a small video, but it is too big for upload here. You can DL it from DropBox here https://www.dropbox.com/s/ah3y93jx9zxv88o/2022-06-04%2008-13-18.mkv?dl=0

If applicable, add save files (zipped), screenshots, or log files to help explain the issue. You can find them in your Documents (`%userprofile%/Documents/CaptainOfIndustry/`). You might need to ZIP the attached files.
[Bug Report.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/8837725/Bug.Report.zip)

**Comment by MaFi-Games** [comment]
Cannot reproduce this, maybe unity update fixed it.

**Comment by gardir** [comment]
I am experiencing this (again?), in Version: Update 1 | v0.5.3e (b225). 

I have attached a savegame in which I've got a building under production and was able to reproduce the "jump to search bar" according to the OP. The amendments I have is the building has to be under construction (not completed), it only happens on '<Left> (A)', and any priority can be selected. It also happens on selecting different build-queue-priority.

[jump-to-search-on-build-priority-select.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/12269813/jump-to-search-on-build-priority-select.zip)
